lichess-bot.py

```diff
@@ -1041,16 +1043,25 @@ def start_lichess_bot() -> None:
     parser.add_argument("--config", help="Specify a configuration file (defaults to ./config.yml).")
     parser.add_argument("-l", "--logfile", help="Record all console output to a log file.", default=None)
     parser.add_argument("--disable_auto_logging", action="store_true", help="Disable automatic logging.")
+    # parser.add_argument("--iter_num", help="Describes which iteration of the model training we are on.")
+    parser.add_argument("--weight_file", help="Describes which iteration of the model training we are on.")
     args = parser.parse_args()
 
     logging_level = logging.DEBUG if args.v else logging.INFO
     auto_log_filename = None
     if not args.disable_auto_logging:
-        auto_log_filename = "./lichess_bot_auto_logs/recent.log"
+        # auto_log_filename = "./lichess_bot_auto_logs/recent.log"
+        auto_log_filename = "/mnt/data/lichess_bot_eval_part_2/lichess-bot/lichess_bot_auto_logs/recent.log"
```

Comment:
again, don't use absolute paths. what if this was running on a  different machine?